spec/abstractops.html

```diff
@@ -548,51 +548,54 @@ <h1>
       GetTemporalUnitValuedOption (
         _options_: an Object,
         _key_: a property key,
-        _unitGroup_: ~date~, ~time~, or ~datetime~,
-        _default_: ~required~, ~unset~, ~auto~, or a Temporal unit,
-        optional _extraValues_: a List of either Temporal units or ~auto~,
+        _default_: ~required~ or ~unset~,
       ): either a normal completion containing either a Temporal unit, ~unset~, or ~auto~, or a throw completion
     </h1>
     <dl class="header">
       <dt>description</dt>
-      <dd>It attempts to read from the specified property of _options_ a Temporal unit that is <emu-not-ref>covered</emu-not-ref> by the union of _unitGroup_ and _extraValues_, substituting _default_ if the property value is *undefined*.</dd>
+      <dd>It attempts to read a Temporal unit from the specified property of _options_.</dd>
     </dl>
     <p>Both singular and plural unit names are accepted, but only the singular form is used internally.</p>
     <emu-alg>
-      1. Let _allowedValues_ be a new empty List.
+      1. Let _allowedStrings_ be « *"auto"* ».
       1. For each row of <emu-xref href="#table-temporal-units"></emu-xref>, except the header row, in table order, do
-        1. Let _unit_ be the value in the "Value" column of the row.
-        1. If the "Category" column of the row is ~date~ and _unitGroup_ is ~date~ or ~datetime~, append _unit_ to _allowedValues_.
-        1. Else if the "Category" column of the row is ~time~ and _unitGroup_ is ~time~ or ~datetime~, append _unit_ to _allowedValues_.
-      1. If _extraValues_ is present, then
-        1. Set _allowedValues_ to the list-concatenation of _allowedValues_ and _extraValues_.
+        1. Let _singularName_ be the value in the "Singular property name" column of the row.
+        1. Append _singularName_ to _allowedStrings_.
+        1. Let _pluralName_ be the value in the "Plural property name" column of the row.
+        1. Append _pluralName_ to _allowedStrings_.
+      1. NOTE: For each singular Temporal unit name that is contained within _allowedStrings_, the corresponding plural name is also contained within it.
```

Comment:
Since we no longer need to build the list dynamically, it might be clearer to write something like

1. Let _allowedStrings_ be a List of all the values in the "Singular property name" and "Plural property name" columns of Table ___.
1. Append **"auto"** to _allowedStrings_.

Or even write it out in full:

1. Let _allowedStrings_ be « **"auto"**, **"year"**, **"years"**, **"month"**, **"months"**, ... ».

(Although writing it out is verbose and repetitive for not really any benefit)

Reply:
Done